### ISSUE TITLE
Add documentation for "scope" in JS adapter init options

### ISSUE THREAD

**Issue by kv979w** [keycloak/keycloak#19146]

**Comment by Ines-Belhaj** [comment]
@kv979w  @jonkoops Indeed, I have read the documentation, and it lacks some specifications related to the scope part in the JS adapter section. So, I took the liberty and added more details to describe the code you provided. You can check my pull request. You can find it in the following link: https://github.com/keycloak/keycloak/pull/20337
